src/lib/__tests__/StateService.test.ts

```diff
@@ -97,6 +105,71 @@ describe('StateService', () => {
     });
   });
 
+  describe('getAdminSettings', () => {
+    let oldMergeSettings: MergeSettings | undefined;
+    beforeEach(() => {
+      oldMergeSettings = config.mergeSettings;
+    });
```

Comment:
Nit: `oldMergeSettings` can probably just be set once in `before` and made a `const`.